bin/githubchart

```diff
@@ -56,7 +60,7 @@ end.parse!
 
 SVG_Path = ARGV.shift
 
-Chart = GithubChart.new(options).svg
+Chart = GithubChart.new(options).send(draw_method)
```

Comment:
Shouldn't this just be supported as an option, vs being sent as a separate call?